server/adapter.go

```diff
@@ -21,16 +21,16 @@ import (
 	"github.com/casbin/casbin/persist"
 	"github.com/casbin/casbin/persist/file-adapter"
 	"github.com/casbin/gorm-adapter"
-	//_ "github.com/jinzhu/gorm/dialects/mssql"
-	//_ "github.com/jinzhu/gorm/dialects/mysql"
-	//_ "github.com/jinzhu/gorm/dialects/postgres"
+	_ "github.com/jinzhu/gorm/dialects/mssql"
```

Comment:
I think just keeping them commented would be better. Anyone who wants to use one of the DBs just can uncomment the corresponding line.